bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js

```diff
@@ -108,7 +108,6 @@ pimcore.object.tags.advancedManyToManyObjectRelation = Class.create(pimcore.obje
         var visibleFields = this.visibleFields;
 
         var columns = [];
-        columns.push({text: 'ID', dataIndex: 'id', width: 50});
```

Comment:
are there any noticable side effects if there are no columns at all (i.e. user error) :-)

Reply:
I tested it. I do not see any problems - does not look beautiful but is exactly what is to be expected when there are neither visible fields nor meta fields:
<img width="1263" alt="Bildschirmfoto 2020-01-30 um 11 36 30" src="https://user-images.githubusercontent.com/8749138/73442330-eebe8a80-4354-11ea-8407-b0d0d0e6bb6f.png">
I added 1 object in this case.

Or should we implement a fallback if the edit grid has no columns then at least show ID (or path)?

Reply:
Fallback sounds reasonable because otherwise you cannot distinguish the rows.

Reply:
Copied fallbacks from many-to-many object relation (except class because advanced m2m object relations can only have one class).

Reply:
so now it is always there ?

Reply:
??? No, the fallback fields get hidden of visibleFields are set:
`{text: 'ID', dataIndex: 'id', width: 50, hidden: !!visibleFields.length},`

The interesting part is `hidden: !!visibleFields.length`

Reply:
But hidden means that you can make it visible again using the column dropdown.
Is it what we really want ?

![image](https://user-images.githubusercontent.com/6142086/73459672-8df47980-4377-11ea-9232-dde6cfd43b7f.png)

Reply:
Ok, then it is also wrong for many-to-many object relation ;-)

I will change the PR to only add those fallback columns if no visible fields have been set...

Reply:
Now the fallback columns are only added if no visible fields have been set.